tutorindigo/templates/indigo/lms/templates/static_templates/about.html

```diff
@@ -0,0 +1,20 @@
+<%namespace name='static' file='../static_content.html'/>
+<%page expression_filter="h"/>
+<%! from django.utils.translation import gettext as _ %>
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("About")}</%block>
+
+<main id="main" aria-label="Content" tabindex="-1">
+    <section class="container about">
+        <h1>
+            <%block name="pageheader">${page_header or _("About")}</%block>
+        </h1>
+        <p>
+            <%block name="pagecontent">${page_content or _("This page left intentionally blank. Feel free to add your own content.")}</%block>
+        </p>
+        <div class="graphic-img">
+            <img src="${static.url('images/about.svg')}" alt="Runs on Tutor" />
+        </div>
```

Comment:
kindly add in comments what have been change in this file as comparable to LMS `footer.html`?